src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleFieldSymbol.vb

```diff
@@ -116,6 +116,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 _tupleElementIndex = other._tupleElementIndex AndAlso
                 TypeSymbol.Equals(_containingTuple, other._containingTuple, TypeCompareKind.ConsiderEverything)
         End Function
+
+        Public NotOverridable Overrides ReadOnly Property IsRequired As Boolean
+            Get
+                Return False
```

Comment:
>Return False [](http://example.com/codeflow?start=16&length=12)

This doesn't look like a justified assumption. The ```_underlyingField``` is almost never a source field